tests/schemas/feature.json

```diff
@@ -0,0 +1,10 @@
+{
+    "type": "object",
+    "properties": {
+        "state": {"type": "string"},
```

Comment:
there are only few valid states, can we have an enum here instead of a string?

Reply:
Looks like it's doable:

> "default": {"type": "string", "enum": ["block", "ignore"]},

(looking at Jason's work here: https://dub.duckduckgo.com/duckduckgo/tracker-radar-internal/blob/master/tests/schema.js )